VMEncryption/main/handle.py

```diff
@@ -628,6 +628,16 @@ def enable_encryption():
                        level=CommonVariables.WarningLevel)
             hutil.redo_last_status()
             exit_without_status_report()
+    
+    cutil = CheckUtil(logger)
```

Comment:
Please add this check to the fatal checks method in CheckUtil instead.

Reply:
I'm talking about the method already called at line 534 of this file.

Reply:
if (volume_type is OS/ALL and OSdiskStatus is NOtEncrypted (ref: line 117) and operation is EnableType (ref: line 141) )
    do the memory check ()